float64 startTime
float64 elapsedTime
float64 speed
Arc arc
GPSCoord lineStartPoint
GPSCoord endPoint
GPSVelocity endVelocity